modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -190,28 +190,28 @@ The chart on the *Application Network* tab shows the xref:visualizer::troublesho
 The following charts appear on the *Overview* dashboard:
 
 Inbound - Total Requests by Response Type::
-The sum of inbound requests, grouped by type of response in automatic intervals. Responses are `OK` or `Failed`. If a Mule app has more than one connector type, all types are included in this metric.
+The sum of inbound requests, grouped by type of response by automatic interval. Responses are `OK` or `Failed`. If a Mule app has more than one connector type, all types are included in this metric.
 
 Inbound - Average Response Time::
-The average response time of inbound requests, grouped in automatic intervals. If a Mule app has more than one connector type, all types are included in this metric.
+The average response time of inbound requests, grouped by automatic interval. If a Mule app has more than one connector type, all types are included in this metric.
 
 Mule Messages::
-The total number of Mule messages that are processed successfully, grouped in automatic intervals.
+The total number of Mule messages that are processed successfully, grouped by automatic interval.
```

Comment:
ditto

Maybe we should include a "For example, ..." in the first text like this, clarifying how the metric appears "grouped by automatic intervals."